x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -312,6 +314,15 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
             });
           }
 
+          if (!isMachineLearningParams(params)) {
```

Comment:
I might have missed it, but should we be checking for EQL rule type or Lucene type query here as well to not run the query for EQL?

Reply:
yes, I check it inside `getWildcardFields`: 
https://github.com/elastic/kibana/pull/157981/files#diff-22cc7a2ca777c08a5aa88de74708d6ff6467b21f13e8a2f4df1c1b42eb5897f8R24 
and have it covered with tests: 
https://github.com/elastic/kibana/pull/157981/files#diff-7c31fa2e8634c1cb7965d4b27f8bc2b0c9e427fdfecdd403b09be483ae46fdd0R33-R50